### ISSUE TITLE
MacOS Build

### ISSUE THREAD

**Issue by CreateTheImaginable** [Autodesk/Aurora#31]
Has anyone tried a MacOS build of HDAurora?    I am curious if any progress has been made on a MacOS build?

Either a Apple Silicon Build or something that uses Rosetta 2 for Intel emulation?

**Comment by pixnblox** [comment]
Yes, we are looking into this! It is possible to _build_ for macOS, but it won't _run_ without some new code. Unfortunately there is no emulation for the ray tracing APIs, i.e. DirectX Raytracing or Vulkan Ray Tracing, and Aurora uses those APIs. So this will require a backend (wrapper) for the Metal Ray Tracing API.
